ui.frontend/src/constants.js

```diff
@@ -42,6 +42,48 @@ export const Constants = {
      */
     FORM_CONTAINER_DATA_ATTRIBUTE: "adaptiveformcontainerPath",
 
+    /**
+     * data attribute to mark the dragged component valid or invalid.
+     * value true for valid
+     * value false for invalid
+     */
+    DATA_ATTRIBUTE_VALID : "data-cmp-valid",
```

Comment:
data-cmp-adaptive-form-valid. We need to follow convention in all data attributes as done in other components.